src/Xenial.Tasty.Core/Reporters/Reporters.ColorScheme.cs

```diff
@@ -24,20 +24,20 @@ private static void SetupConsoleEncoding()
             Console.InputEncoding = Encoding.UTF8;
         }
 
-        public ConsoleColor DefaultColor = ConsoleColor.White;
-        public ConsoleColor ErrorColor = ConsoleColor.Red;
-        public ConsoleColor WarningColor = ConsoleColor.Yellow;
-        public ConsoleColor NotifyColor = ConsoleColor.DarkGray;
-        public ConsoleColor SuccessColor = ConsoleColor.DarkGreen;
-
-        public string ErrorIcon = "👎";
-        public string SuccessIcon = "👍";
-        public string NotRunIcon = "🙈";
-        public string IgnoredIcon = "🙄";
-
-        public static ColorScheme Default = SupportsRichContent()
-            ? new ColorScheme()
-            : new ColorSchemeLegacy();
+        public ConsoleColor DefaultColor { get; set; } = ConsoleColor.White;
+        public ConsoleColor ErrorColor { get; set; } = ConsoleColor.Red;
+        public ConsoleColor WarningColor { get; set; } = ConsoleColor.Yellow;
+        public ConsoleColor NotifyColor { get; set; } = ConsoleColor.DarkGray;
+        public ConsoleColor SuccessColor { get; set; } = ConsoleColor.DarkGreen;
+
+        public string ErrorIcon { get; set; }= "👎";
+        public string SuccessIcon { get; set; } = "👍";
+        public string NotRunIcon { get; set; } = "🙈";
+        public string IgnoredIcon { get; set; } = "🙄";
+
+        public static ColorScheme Default => SupportsRichContent()
```

Comment:
That would break the configurability cause this will create a new ColorScheme every time you access the `Default` property. So this should be a readonly getter:

```cs
public static ColorScheme Default { get; } = SupportsRichContent()
```

Reply:
It is very early for me, but wasn't it what it was doing before? Not storing the scheme in a field and returning a new one every time it is accessed.

Reply:
No a field initializer runs in the `ctor`. This syntax does the same and lowers under the hood to

```cs
static ColorScheme _Default = new ColorScheme ....;
public static ColorScheme Default {
get 
{
return _Default;
}
}

Reply:
Corrected in 5fbda570b0b835965f7aa1dd3a7070bd5857e8d8